### PR TITLE
Add rpm-validation pipelines trigger to opensearch and dashboards distribution builds

### PR DESCRIPTION
**Pull request by peterzhuamazon**
Signed-off-by: Peter Zhu <zhujiaxi@amazon.com>

### Description
Add rpm-validation pipelines trigger to opensearch and dashboards distribution builds
 
### Issues Resolved
Part of #1555
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: jenkins/opensearch-dashboards/distribution-build.jenkinsfile [hunk 1]
```diff
@@ -173,6 +173,16 @@ pipeline {
                             echo "buildManifestUrl (x64, rpm): ${buildManifestUrl}"
                             echo "artifactUrl (x64, rpm): ${artifactUrl}"
 
+                            String bundleManifestUrl = buildManifestObj.getBundleManifestUrl(JOB_NAME, BUILD_NUMBER)
+
+                            echo "Trigger rpm validation for x64 rpm ${buildManifestUrl}"
```
Comment (zelinh) [inline jenkins/opensearch-dashboards/distribution-build.jenkinsfile:178]: This should be `bundleManifestUrl`.

File: jenkins/opensearch-dashboards/distribution-build.jenkinsfile [hunk 2]
```diff
@@ -347,6 +357,16 @@ pipeline {
                                     echo "buildManifestUrl (arm64, rpm): ${buildManifestUrl}"
                                     echo "artifactUrl (arm64, rpm): ${artifactUrl}"
 
+                                    String bundleManifestUrl = buildManifestObj.getBundleManifestUrl(JOB_NAME, BUILD_NUMBER)
+
+                                    echo "Trigger rpm validation for arm64 rpm ${buildManifestUrl}"
```
Comment (zelinh) [inline jenkins/opensearch-dashboards/distribution-build.jenkinsfile:362]: ^^^

File: jenkins/opensearch-dashboards/distribution-build.jenkinsfile (hunk 1 above)
Comment (peterzhuamazon) [reply jenkins/opensearch-dashboards/distribution-build.jenkinsfile]: fixed now.

**Review by zelinh — APPROVED** [review]
Thanks.

**Review by bbarani — APPROVED** [review]

**Pull request merged**